basic_nodes/src/main/java/club/mondaylunch/gatos/basicnodes/ValueProviderNodeType.java

```diff
@@ -0,0 +1,47 @@
+package club.mondaylunch.gatos.basicnodes;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import club.mondaylunch.gatos.core.data.DataBox;
+import club.mondaylunch.gatos.core.data.DataType;
+import club.mondaylunch.gatos.core.graph.connector.NodeConnector;
+import club.mondaylunch.gatos.core.graph.type.NodeType;
+
+public class ValueProviderNodeType<T> extends NodeType.Start {
```

Comment:
This should actually be a process node, as it doesn't trigger the start of a flow.